app/controllers/ems_common.rb

```diff
@@ -834,6 +837,8 @@ def get_form_vars
     @edit[:new][:bearer_token] = params[:bearer_token] if params[:bearer_token]
     @edit[:new][:bearer_verify] = params[:bearer_verify] if params[:bearer_verify]
 
+    @edit[:new][:service_account] = params[:service_account] if params[:service_account]
+
```

Comment:
@lwander, keep in mind that the new field that you add for `service_account_json` should also be encrypted like the [`password` field is](https://github.com/ManageIQ/manageiq/blob/2b977353422ddd5884ff55840922e9319cd60404/app/models/authentication.rb#L13).